site/en/docs/crux/methodology/index.md

```diff
@@ -0,0 +1,567 @@
+
+Popularity is provided as a guide for broad analysis, e.g. to determine performance by country for the top 1,000 origins. It should not be used to determine an origin's overall popularity on the web.
+
+#### Notification Permissions {: #notification-permissions-metric }
+
+{% Aside %}
+"The Notifications API allows web pages to control the display of system notifications to the end user. These are outside the top-level browsing context viewport, so therefore can be displayed even when the user has switched tabs or moved to a different app. The API is designed to be compatible with existing notification systems, across different platforms."
+
+<cite><a href="https://developer.mozilla.org/docs/Web/API/Notifications_API">MDN</a></cite>
+{% endAside %}
+
+## Dimensions
+
+The CrUX dataset includes dimension data to allow deeper interrogation of the data. Dimensions identify a specific group of data that a record is being aggregated against, e.g. a form factor of `"phone"` indicates that the record contains information about loads that took place on a mobile device.
+
+Data may not be available for all dimensions, based on [eligibility criteria](#page-eligibility).
+
+### Effective Connection Type {: #ect-dimension }
+
+[Effective Connection Type](https://developer.mozilla.org/docs/Glossary/Effective_connection_type) (ECT) is a web platform API to broadly categorize visitor connection speeds. This dimension in the CrUX dataset allows you to:
+
+- See a breakdown of connection speeds of real visitors
+- Filter performance data by connection speed
+
+Note that the specification defines four connection types, the majority of visits will likely be on connections faster than `3G` and thus be classified as `4G`:
+
+<div class="responsive-table">
+<table class="with-heading-tint width-full fixed-table">
+<thead>
+<tr>
+<th>ECT</th>
+<th>Minimum RTT</th>
+<th>Maximum downlink</th>
+<th>Explanation</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>offline</td>
+<td>N/A</td>
+<td>N/A</td>
+<td>The network is offline, only cached files can be served.</td>
+</tr>
+<tr>
+<td>slow-2g</td>
+<td>2000ms</td>
+<td>50 Kbps</td>
+<td>The network is suited for small transfers only such as text-only pages.</td>
+</tr>
+<tr>
+<td>2g</td>
+<td>1400ms</td>
+<td>70 Kbps</td>
+<td>The network is suited for transfers of small images.</td>
+</tr>
+<tr>
+<td>3g</td>
+<td>270ms</td>
+<td>700 Kbps</td>
+<td>The network is suited for transfers of large assets such as high resolution images, audio, and SD video.</td>
+</tr>
+<tr>
+<td>4g</td>
+<td>0ms</td>
+<td>∞</td>
+<td>The network is suited for HD video, real-time video, etc.</td>
+</tr>
+</tbody>
+</table></div>
+
+### Form Factor {: #form-factor-dimension }
+
+The form factor dimension allows you to query against three separate form factors:
+
+- Mobile (`PHONE`)
```

Comment:
Why in Uppercase?

Reply:
That's how it appears in the CrUX API. Perhaps we don't need to use that convention outside of the API docs though. WDYT @simonhearne?

Reply:
IMHO it should be in uppercase since it's in code format. Though there's a couple of other places we probably shoudl correct this. Will make comments.

Reply:
I don't think it's necessary to have the content in brackets at this point in the docs, I'm in favour of removing them and simplifying to :
- Phone
- Tablet
- Desktop